jerry-core/jmem/jmem.h

```diff
@@ -199,7 +199,7 @@ void jmem_run_free_unused_memory_callbacks (jmem_free_unused_memory_severity_t s
   { \
     JERRY_ASSERT (var_name ## ___size != 0); \
     \
-    jmem_heap_free_block (var_name, var_name ## ___size); \
+    jmem_heap_free_block ((void *) (var_name), var_name ## ___size); \
```

Comment:
This is something that certainly should _not_ be changed. Same comments hold as for the change above. If anything is to be changed then it's line 1261 of ecma-regexp-object.c, where `saved_p` is declared as a local array of `const lit_utf8_byte_t *` elements. (MSVC reports the warning for line 1428, but 1261 is the root cause.) Again, I'm not sure why MSVC considers this a problem.